### PR TITLE
Updated Translations and Widget Preview

### PR DESCRIPTION
**Pull request by user0-07161**
Updated some german translations and updated widget preview to show that there is a dark mode for the widget.

Widget Preview:

WidgetPreview

### CODE REVIEW

**Comment by SapuSeven** [comment]
Hi,
Thanks for your interest in improving BetterUntis.

Please take a look at the [Contributing-Section of the Readme file](https://github.com/SapuSeven/BetterUntis#contributing).
It explains how to edit translations and where to open a pull request. I cannot merge your changes into `master`.

Regarding the updated preview Image: You seem to have mistakenly removed the alpha channel.
If you look at the original image, you can see a transparent border around the image - your edit missed that.
Also I feel that the dark background is too dark - take a look at the actual app with the original red color theme in dark mode and pick the dark color from there.
Lastly, I see some artifacts around parts of the grey line on top of the dark background.

Ultimately, I'm not too sure about this idea. I looked at other apps and for example the Google Calendar app manages to have different colored previews for light and dark mode: Maybe we could have a look at how they did it.
Screenshot_20230723-125735
Screenshot_20230723-125721

**Comment by user0-07161** [comment]
Hello SapuSeven,

in [https://developer.android.com/develop/ui/views/appwidgets/enhance#dynamic-colors](https://developer.android.com/develop/ui/views/appwidgets/enhance#dynamic-colors) it says that as of android 12 it is possible to create different previews for dark/light mode. Therefore, many people are still using an older android version, I think it will not be possible to add this in BetterUntis.

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -276,7 +276,7 @@
     <string name="loading">Wird geladen</string>
     <string name="loading_failed">Ladevorgang fehlgeschlagen</string>
     <string name="settings_dialog_url_open_title">Keine kompatible App gefunden</string>
-    <string name="settings_dialog_url_open_text">BetterUntis konnte keine App finden, die mit dem Öffnen von Links umgehen kann.
+    <string name="settings_dialog_url_open_text">BetterUntis konnte keine App finden, die mit dem Links öffnen kann.
```
Comment (lm41) [inline app/src/main/res/values-de/strings.xml]: This sentence is grammaticly not correct. Please provide a valid german sentence.

**Review by lm41 — CHANGES_REQUESTED** [review]

**Comment by lm41** [comment]
https://developer.android.com/develop/ui/views/appwidgets/enhance#backward-compatibility

There is backwards compability.

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (user0-07161) [reply app/src/main/res/values-de/strings.xml]: Sorry, i mean "BetterUntis konnte keine App finden, die mit Links öffnen kann."

Comment (lm41) [reply app/src/main/res/values-de/strings.xml]: I would prefer: "BetterUntis konnte keine App finden, die Links öffnen kann."

Comment (user0-07161) [reply app/src/main/res/values-de/strings.xml]: My device make (whyever) autocorrect. Your Version is better.

**Review by lm41 — COMMENTED** [review]
The strings are fine now. I still think that the m3/you preview should be used. Regarding the I would suggest to close this pr and edit them on weblate.

**Comment by lm41** [comment]
See https://github.com/SapuSeven/BetterUntis#translating-into-another-language

**Comment by SapuSeven** [comment]
Given the feedback I will close this PR.
Translations can be done on weblate (see @lm41 's comment), and I agree that the app should take advantage of the modern dynamic widget preview approach. If you want to work on it, feel free to open another PR, and keep the README instructions in mind ;)

**Pull request closed without merging**